forms-flow-api/src/formsflow_api/resources/public_endpoints.py

```diff
@@ -282,3 +283,29 @@ def get():
             HTTPStatus.OK,
         )
         return response, status
+
+
+# todo: do we need any other decorators, check with the team or request for PR review?# pylint: disable=fixme
+@cors_preflight("POST,OPTIONS")
+@API.route("/<string:definition_key>/start", methods=["POST", "OPTIONS"])
```

Comment:
Can we convert this into an authorized API instead of keeping it public, and have Keycloak call it using a tenant-specific client ID and secret?

Reply:
This endpoint is triggered during the registration process for the try-it-now client, Once registered, the user is disabled and will not be logged in. for this tenant (try-it-now) client only registration will be enabled. login is always disabled. I will add comments aswell